ui/jobs/components/index.ts

```diff
@@ -376,4 +357,33 @@ export class ComponentManager {
     )
       this.bars.setJobsContainerVisibility(false);
   }
+
+  private _shouldShowFoodBuff(): void {
```

Comment:
Could you call this `_updateFoodBuff` or `_updateFoodBuffDisplay` or something?

Mostly my preference is that if a function starts with "should" then it's a small function that returns a boolean, e.g. shouldShowFoodBuff returns true or false, whether or not it should show.  And so the naming is a little bit confusing to me.